code-scanning/frogbot-scan-and-fix.yml

```diff
@@ -18,6 +18,7 @@ on:
 permissions:
   contents: write
   pull-requests: write
+  security-events: write
```

Comment:
Why is the permission change needed now?

Reply:
Hi @Phantsure, it is only needed in the scan-and-fix, because we only use the code scanning upload in scan-and-fix.